lib/common/dsr.c

```diff
@@ -85,22 +93,29 @@ int h2o_dsr_parse_req(h2o_dsr_req_t *req, const char *_value, size_t _value_len,
     const char *name;
     size_t name_len;
     int64_t n;
+    struct st_h2o_dsr_req_quic_t quic_req;
 
-    req->quic_version = 0;
-    req->cipher = 0;
-    req->address.sa.sa_family = AF_UNSPEC;
+    memset(req, 0, sizeof(*req));
+    memset(&quic_req, 0, sizeof(quic_req));
+    req->transport.quic.address.sa.sa_family = AF_UNSPEC;
```

Comment:
```c
struct st_h2o_dsr_req_quic quic_req = {.address.sa.sa_family = AF_UNSPEC};

*req = (h2o_dsr_req_t){};
```

Now that we have `quic_req`, I think the address of that structure should be set to `AF_UNSPEC`. And we can use compound assignment for the purpose.